proxyd/integration_tests/sender_rate_limit_test.go

```diff
@@ -57,8 +57,16 @@ func TestSenderRateLimitValidation(t *testing.T) {
 		require.NoError(t, err)
 		t.Run(name, func(t *testing.T) {
 			res, _, err := client.SendRequest([]byte(body))
-			require.NoError(t, err)
-			RequireEqualJSON(t, []byte(expResponseBody), res)
+
+			// An update to geth now causes this case to fail.
+			// Legacy transactions are now disabled by default on the modern signer: https://github.com/ethereum/go-ethereum/pull/31434
+			if strings.Contains(name, "no chain id (pre eip-155)") {
+				require.Error(t, err)
+				return
```

Comment:
We should manually enable legacy txs again. Otherwise the deterministic deployment txs that people use, which don't have a chainID for cross-chain compat, won't be possible to confirm anymore.